python/whylogs/api/whylabs/session_manager.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import os
+import uuid
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional, Union
+
+from whylabs_client.api.sessions_api import (
+    CreateSessionRequest,
+    CreateSessionResponse,
+    SessionsApi,
+)
+from whylabs_client.api_client import ApiClient, Configuration
+
+from whylogs.api.whylabs.auth_file import get_auth_file_path
+from whylogs.api.whylabs.notebook_check import is_notebook
+from whylogs.api.whylabs.variables import Variables
+
+DEFAULT_PATH = os.getenv("WHYLOGS_CONFIG_PATH") or f"{Path.home()}/.whylabs/auth.ini"
+_auth_path = get_auth_file_path(auth_path=Path(DEFAULT_PATH))
+
+DEFAULT_WHYLABS_HOST = "https://api.whylabsapp.com"
```

Comment:
consider:
```suggestion
DEFAULT_WHYLABS_HOST = os.environ.get("WHYLABS_API_ENDPOINT") or "https://api.whylabsapp.com"
```

Consider moding this into a common file so we can use same definition for session manager and the whylabs writer.

Reply:
ok! makes sense